scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -150,7 +150,9 @@ package object dialects {
 
   implicit val Scala32 = Scala31
 
-  implicit val Scala3 = Scala32
+  implicit val Scala33 = Scala31.withAllowFewerBraces(true)
```

Comment:
and i was recently wondering whether we needed to add this to the future dialect... thank you!

Reply:
ah, wait, it should be Scala32.withXxx.

Reply:
Ach my bad, should be fixed now.